apps/api/src/app/events/e2e/bridge-trigger.e2e.ts

```diff
@@ -626,62 +626,6 @@ contexts.forEach((context: Context) => {
       expect(sentMessage[1].subject).to.include('prefix Hello default_name');
       expect(sentMessage[0].subject).to.include('prefix Hello payload_name');
     });
-    it(`should trigger the bridge workflow with control default and payload data [${context.name}] - with backwards compatability for payload variable`, async () => {
```

Comment:
Backwards compatibility is now dropped for usage of `payload` variables in LiquidJS without using the `payload` namespace.